client/vite.config.js

```diff
@@ -24,7 +24,11 @@ const config = ({ mode }) => {
       host: VITE_HOST,
       port: Number(VITE_PORT),
       proxy: {
-        [VITE_API_PATH]: VITE_API_SERVER
+        [VITE_API_PATH]: VITE_API_SERVER,
+        [VITE_SOCKET_PATH]: {
```

Comment:
Did you have a chance to test it? I have some difficulties with it on my project and the config has the next view? but if your configuration is suitable, then will stay with it
<img width="285" alt="image" src="https://github.com/BinaryStudioAcademy/thread-js/assets/43109766/9c83fbbe-571a-409b-b1f9-156c66ad9a71">

Reply:
Yes, i've tested it and it works correct. 
But i've encountered an issue with leaving reaction on post - toast message doesn't appear. I will create a separate task for this.